index.html

```diff
@@ -118,18 +124,20 @@ <h3 class="widget-title">Become a member!</h3>
         </div>
         <!--End mc_embed_signup-->
 
-        <!-- Twitter feed -->
-        <aside id="twitterwidget" class="twitter_widget">
-        <a class="twitter-timeline" href="https://twitter.com/TRUCSClub" data-widget-id="546200035717681152" data-chrome="nofooter">Tweets from @TRUCSClub</a>
-          <script>
-                !function(d,s,id){var js,fjs=d.getElementsByTagName(s)[0],p=/^http:/.test(d.location)?'http':'https';if(!d.getElementById(id)){js=d.createElement(s);js.id=id;js.src=p+"://platform.twitter.com/widgets.js";fjs.parentNode.insertBefore(js,fjs);}}(document,"script","twitter-wjs");
-          </script>
+        
         </aside>
 			</div><!-- /Sidebar -->
 		</div>
     </div>
     
-    <footer class="footer"><p>Copyright 2014 Computer Science Club</p></footer>
+    <footer class="footer"><p>Copyright &copy;
+        <script language="javascript" type="text/javascript">
```

Comment:
Just FYI, in HTML5 if you specify `<script>` then `language="javascript"` and `type="text/javascript"` are automatically assumed, so it's still valid to leave them out. What you have works as well though of course :)